src/stdbuf/stdbuf.rs

```diff
@@ -1,5 +1,4 @@
 #![crate_name = "stdbuf"]
-#![feature(negate_unsigned)]
```

Comment:
doesn't seem to be strictly needed, but there are still warnings that should be fixed IMO